incubator/hnc/test/e2e/issues_test.go

```diff
@@ -225,4 +225,15 @@ var _ = Describe("Issues", func() {
 		runShouldNotContain("CannotPropagateObject", 1, "kubectl hns describe", nsParent)
 		runShouldNotContain("CannotUpdateObject", 1, "kubectl hns describe", nsChild)
 	})
+
+	It("Should propogate admin rolebindings - issue 772", func(){
+		// set up
+		mustRun("kubectl create ns", nsParent)
+		mustRun("kubectl create ns", nsChild)
+		mustRun("kubectl hns set", nsChild, "--parent", nsParent)
+		// Creating admin rolebinding object and waiting 2s
```

Comment:
Remove wait 2s in the comment. Does `mustRun` have eventually?

Reply:
Yes. Done